assets/heatmap_header.txt

```diff
@@ -0,0 +1,12 @@
+#id: 'Sample Similarity'
+#title: 'Sample Similarity' 
+#section_name: 'edgeR: Sample Similarity' 
+#description: "is generated from normalised gene counts through
+#       <a href='https://bioconductor.org/packages/release/bioc/html/edgeR.html' target='_blank'>edgeR</a>.
+#       Euclidean distances between log<sub>2</sub> normalised CPM values are then calculated and clustered."
+#plot_type: 'heatmap'
+#anchor: 'ngi_rnaseq-sample_similarity'
+#pconfig:
+#    title: 'edgeR: Euclidean distances'
+#    xlab: True
+#    reverseColors: 'True'
```

Comment:
OCD inconsistency about using quotes here and above. I'd drop the quotes for this personally as it's not really a string.